mysql-wordpress-pd/README.md

```diff
@@ -4,367 +4,207 @@
-  kubectl get secret mysql-pass && \
-  kubectl get pv
-```
-
-```shell
-NAME                     DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-deploy/wordpress-mysql   1         1         1            1           3m
-NAME                                  READY     STATUS    RESTARTS   AGE       IP           NODE
-po/wordpress-mysql-3040864217-40soc   1/1       Running   0          3m        172.17.0.2   127.0.0.1
-NAME                  CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE       SELECTOR
-svc/wordpress-mysql   None         <none>        3306/TCP   3m        app=wordpress,tier=mysql
-NAME                 ENDPOINTS         AGE
-ep/wordpress-mysql   172.17.0.2:3306   3m
-NAME                 STATUS    VOLUME       CAPACITY   ACCESSMODES   AGE
-pvc/mysql-pv-claim   Bound     local-pv-2   20Gi       RWO           3m
-NAME         TYPE      DATA      AGE
-mysql-pass   Opaque    1         3m
-NAME         CAPACITY   ACCESSMODES   STATUS      CLAIM                    REASON    AGE
-local-pv-1   20Gi       RWO           Available                                      3m
-local-pv-2   20Gi       RWO           Bound       default/mysql-pv-claim             3m
-```
+The following manifest describes a single-instance MySQL Deployment. The MySQL container mounts the PersistentVolume at /var/lib/mysql. The `MYSQL_ROOT_PASSWORD` environment variable sets the database password from the Secret. 
+
+{% include code.html language="yaml" file="mysql-deployment.yaml" ghlink="/docs/tutorials/stateful-application/mysql-deployment.yaml %}
+
+1. Deploy MySQL from the `mysql-deployment.yaml` file:
+
+       kubectl create -f mysql-deployment.yaml
+
+2. Verify that the Pod is running by running the following command:
+
+       kubectl get pods
+
+   **Note:** It can take up to a few minutes for the Pod's Status to be `RUNNING`.
+   {: .note}
+
+   The response should be like this:
+
+       NAME                               READY     STATUS    RESTARTS   AGE
+       wordpress-mysql-1894417608-x5dzt   1/1       Running   0          40s
 
 ## Deploy WordPress
 
-Next deploy WordPress using
-[wordpress-deployment.yaml](https://git.k8s.io/examples/mysql-wordpress-pd/wordpress-deployment.yaml):
+The following manifest describes a single-instance WordPress Deployment. It uses many of the same features like a PVC for persistent storage and a Secret for the password. But it also uses a different setting: `"type": NodePort`. This setting exposes WordPress to traffic from outside of the cluster.
+
+{% include code.html language="yaml" file="mysql-deployment.yaml" ghlink="/docs/tutorials/stateful-application/wordpress-deployment.yaml %}
+
+1. Deploy WordPress from the `wordpress-deployment.yaml` file:
+
+       kubectl create -f wordpress-deployment.yaml
 
-```shell
-kubectl create -f $KUBE_REPO/mysql-wordpress-pd/wordpress-deployment.yaml
-```
+2. Verify that the Service is running by running the following command:
```

Comment:
This sounds odd. You just deployed a Deployment, but querying the Service below. Can you please double check?